packages/plugin-core/package.json

```diff
@@ -723,6 +723,11 @@
         "dendron.serverPort": {
           "type": "number",
           "description": "port for server. If not set, will be randomly generated at startup."
+        },
```

Comment:
thoughts about adding this in `dendron.yml`  instead of the workspace? the reason is that we're working on removing configuration that relies on vscode

Reply:
That wouldn't make sense because the watcher works **outside** Dendron workspaces.

Reply:
ah, you're right. ignore that comment